MaterialComponents.podspec

```diff
@@ -102,29 +104,30 @@ Pod::Spec.new do |mdc|
 
     component.dependency "MDFInternationalization"
     component.dependency "MaterialComponents/Buttons"
-    component.dependency "MaterialComponents/NavigationBar"
+    component.dependency "MaterialComponents/NavigationBar+Extensions"
```

Comment:
Confirming that we are intentionally bringing in the extensions here.

If so, can we bring in a more specific subset of extensions?

Reply:
Unfortunately so: https://github.com/material-components/material-components-ios/blob/develop/components/BottomAppBar/src/MDCBottomAppBarView.m#L101

I will update it to point only to the themer in case more extensions are added in the future.